_maps/RandomRuins/IceRuins/icemoon_underground_lavaland.dmm

```diff
@@ -53,12 +53,6 @@
 "T" = (
 /turf/open/floor/cult,
 /area/icemoon/underground)
-"V" = (
```

Comment:
Didn't even realize that this toolbox was the traitor one but any reason in particular why its being removed all together and not replaced with an alternate toolbox